unified-runtime/scripts/core/exp-graph.yml

```diff
@@ -0,0 +1,214 @@
+  - type: void*
+    name: pNext
+    desc: "[in][optional] pointer to extension-specific structure."
+returns:
+  - $X_RESULT_SUCCESS
+  - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "End graph capture on the specified immediate command list."
+class: $xQueue
+name: EndGraphCaptureExp
+params:
+  - type: $x_queue_handle_t
+    name: hQueue
+    desc: "[in] Handle of the queue on which to end graph capture."
+  - type: $x_exp_graph_handle_t*
+    name: phGraph
+    desc: "[out] Pointer to the handle of the captured graph object."
+  - type: void*
+    name: pNext
+    desc: "[out][optional] pointer to extension-specific structure."
+returns:
+  - $X_RESULT_SUCCESS
+  - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Instantiate an executable graph from a recorded graph."
+class: $xQueue
+name: InstantiateGraphExp
+params:
+  - type: $x_exp_graph_handle_t
+    name: hGraph
+    desc: "[in] Handle of the recorded graph to instantiate."
+  - type: $x_exp_executable_graph_handle_t*
+    name: phExecutableGraph
+    desc: "[out] Pointer to the handle of the instantiated executable graph."
+  - type: void*
+    name: pNext
+    desc: "[out][optional] pointer to extension-specific structure."
+returns:
+  - $X_RESULT_SUCCESS
+  - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Append an executable graph to the queue."
+class: $xQueue
+name: AppendGraphExp
+params:
+  - type: $x_queue_handle_t
+    name: hQueue
+    desc: "[in] Handle of the queue to append the graph to."
+  - type: $x_exp_executable_graph_handle_t
+    name: hGraph
+    desc: "[in] Handle of the executable graph to append."
+  - type: void*
+    name: pNext
+    desc: "[in][optional] pointer to extension-specific structure."
+  - type: $x_event_handle_t
+    name: hSignalEvent
+    desc: "[in][optional] Event to be signaled on completion."
+  - type: uint32_t
+    name: numWaitEvents
+    desc: "[in][optional] Number of events to wait on before executing."
+  - type: $x_event_handle_t*
+    name: phWaitEvents
+    desc: "[in][optional][range(0, numWaitEvents)] Pointer to an array of events to wait on."
+returns:
+  - $X_RESULT_SUCCESS
+  - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Destroy a recorded graph object."
+class: $xGraph
+name: DestroyExp
+params:
+    - type: $x_exp_graph_handle_t
+      name: hGraph
+      desc: "[in] Handle of the graph object to destroy."
+returns:
+    - $X_RESULT_SUCCESS
+    - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Destroy an instantiated executable graph object."
```

Comment:
Mention that `Executable graph instance must not be executing on any queue when calling this function`.